bin/cloudgov/setup-egress-for-apps

```diff
@@ -145,9 +147,22 @@ for app in $applist; do
     $output cf set-env "$app" AWS_CA_BUNDLE /etc/ssl/certs/ca-certificates.crt
     $output cf set-env "$app" REQUESTS_CA_BUNDLE /etc/ssl/certs/ca-certificates.crt
 
+    ### Ensure that our primary apps are all attached to the log drain service!
```

Comment:
We need to ensure that the log-shipper-drain service does _not_ get bound to the log-shipper itself! (So far we seem to have avoided this on the log-shipper in the dr space.) 

There's a little script in the usagov-logshipper repo to do this: bin/add-log-drains-for-space.sh. And there is a circleci action for it in that repo, but of course no one finds it. So I don't object to adding it here, but we should include that clause: 
```    if [[ ! "$app" = "log-shipper" ]]; then``` 

(I've come to believe that we should fold the logshipper code back into the usagov-2021 repo, sigh.)

Reply:
OK, adding conditional here.